builder/test/pyramid.cc

```diff
@@ -128,7 +135,9 @@ TEST(pyramid_multi, pipeline) {
   func upsample = func::make(pyramid_upsample2x,
       {{in, {point(x), point(y)}}, {up1, {bounds(x, x + 1) / 2, bounds(y, y + 1) / 2}}}, {{out, {x, y}}});
 
-  upsample.loops({{y, 1}});
+  if (mode == 1) {
+    upsample.loops({{y, 1}});
```

Comment:
Nit: rename the parameter to split_y and use that as the split factor. There are a ton of other tests that do the same thing, they make the parameter the split factor, where split == 0 means don't split.